rustls/src/common_state.rs

```diff
@@ -883,6 +905,17 @@ enum Limit {
     No,
 }
 
+pub(crate) enum RpkNegotiationResult {
+    Negotiated,
+    NotNegotiated,
+    Err(Error),
+}
+
+pub(super) struct RawKeyNegotationParams {
```

Comment:
I think you have a typo in this type name: `RawKeyNegotationParams` -> `RawKeyNegotiationParams`